v63004/u_inref/gtm8874.csh

```diff
@@ -0,0 +1,76 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+#### testA ####
+# Basic VIEW "[no]STATSHARE":<region-list> functionality
+# Basic VIEW "[no]STATSHARE" functionality
+# VIEW command with disabled regions
+echo "# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG"
+$gtm_tst/com/dbcreate.csh mumps 3 >>& dbcreate_log_1.txt
+
+echo ''
+echo '# Disable sharing for BREG'
+$MUPIP set -NOSTAT  -reg "BREG" #>>& dbcreate_log.txt
+
+echo '# Run testA of gtm8874.m to test basic VIEW functionality and disabled regions'
```

Comment:
Can the description of testA, testB1, testB2 and testC be improved. I am having a hard time understanding what they are testing from this description. For example, by "disabled regions", what is meant as disabled? Also by "VIEW" functionality, do we mean "VIEW STATSHARE" functionality? etc.